Userland/Libraries/LibWeb/HTML/HTMLImageElement.idl

```diff
@@ -21,7 +21,7 @@ interface HTMLImageElement : HTMLElement {
     [FIXME] readonly attribute USVString currentSrc;
     [FIXME, CEReactions] attribute DOMString referrerPolicy;
     [FIXME, CEReactions] attribute DOMString decoding;
-    [FIXME, CEReactions] attribute DOMString loading;
+    [CEReactions, Reflect] attribute DOMString loading;
```

Comment:
[The specification](https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-loading) says that this should be limited to only known values.
In this case, because this is a [lazy loading attribute](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#lazy-loading-attribute), the known values are 'lazy' and 'eager'. The missing value default and invalid value default are both 'eager'.

This can be implemented using the `[Enumerated]` extended attribute. There was a recent PR merged (#24445), which is a good example of this feature in action.

A test showing the attribute can only be set to these values would also be great!

Reply:
Thank you :)

I've made these changes & added unit tests.